microsoft-edge/webview2/release-notes.md

```diff
@@ -32,6 +32,149 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 
+<!-- ====================================================================== -->
+## 1.0.1370.28
+
+Release Date: October 11, 2022
+
+[NuGet package for WebView2 SDK 1.0.1370.28](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1370.28)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 104.0.1370.28 or higher.

```

Comment:
This PR has 104 & 105, is this correct?  Last PRs had (for Release then Prerelease sections):
https://learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes?tabs=dotnetcsharp#10134322
105 & 106
104 & 105
103 & 105
102 & n/a
101 & 102

Reply:
106.0.1370.28 for release and 107.0.1414.0 for prerelease

Reply:
I entered two Suggestion markup comments with that fix, so you can Resolve the present conversation